migrations/Version20201216112814.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20201216112814 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->abortIf('postgresql' !== $this->connection->getDatabasePlatform()->getName(), 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('DROP INDEX uniq_911533c8d1b862b8');
+        $this->addSql('ALTER TABLE conference DROP hash');
```

Comment:
Why `hash` column is dropped here? It looks like an mistake

Reply:
I agree, I don't know what it's doing here, I saw it and thought it may have been done previously and not added to a migration so I let it here, but I can drop it :+1:

Reply:
I merged another PR where he fixed this stuff :)